docs/getting-started/download_models.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Downloading LLMs
+description: Steps on how to download models using `ilab`
+logo: images/ilab_dog.png
+---
+
+# 📥 Download the model
+
+- Run the `ilab model download` command.
+
+```shell
+ilab model download
+```
+
+`ilab model download` downloads a compact pre-trained version of the [model](https://huggingface.co/instructlab/) (~4.4G) from HuggingFace:
+
+```shell
+(venv) $ ilab model download
+Downloading model from Hugging Face: instructlab/merlinite-7b-lab-GGUF@main to /Users/USERNAME/Library/Caches/instructlab/models...
```

Comment:
We should update this to granite right?

Reply:
Im pretty sure the default is still merlinite, I'll change it to granite once that is updated.